python/ray/_private/ray_constants.py

```diff
@@ -581,3 +581,6 @@ def gcs_actor_scheduling_enabled():
 )
 
 RAY_GC_MIN_COLLECT_INTERVAL = env_float("RAY_GC_MIN_COLLECT_INTERVAL_S", 5)
+
+WARN_BLOCKING_GET_INSIDE_ASYNC_ENV_VAR = "WARN_BLOCKING_GET_INSIDE_ASYNC"
```

Comment:
this looks fine, but it needs to be in `ray._common/` instead of `ray._private/` if train is going to rely on it

let's also prefix it with `RAY_`